SU2_PY/OptimalPropeller.py

```diff
@@ -0,0 +1,371 @@
+# Reference   : Glauert H., Airplane Propellers, in Aerodynamic Theory, Ed. Durand W. F.,
+#               Vol. IV, pp. 169 - 360, Springer, 1935.
+# Input       : Interactive.
+# Output      : ActuatorDisk.cfg, containing part of SU2 .cfg file.
+#               ActuatorDisk.dat, containing propeller load distribution to be read by SU2_CFD.
+# Note        : Python 3 or higher needed.
+# ==============================================================================================
+
+import math
+import numpy as np
+import pylab as pl
+
+##########################
+###     Functions      ###
+##########################
+def a_distribution (w0, Chi):
+
+    """Function used to compute the value of the axial interference factor using the inviscid theory of the optimal propeller."""
+
+    a = (w0*pow(Chi,2))/(pow(Chi,2)+pow((1+(w0)),2))
+    return a
+
+def Print_external_file(CTrs, CPrs):
+
+    """Function used to write the actuator disk input data file"""
+    file = open('ActuatorDisk.dat', 'w')
+    file.write('# Automatic generated actuator disk input data file using the Optimal Propeller code.\n')
+    file.write('# Data file needed for the actuator disk VARIABLE_LOAD type.\n')
+    file.write('# The load distribution is obtained using the inviscid theory of the optimal propeller\n')
+    file.write('# using global data.\n')
+    file.write('#\n')
+    file.write('# The first three lines must be filled.\n')
+    file.write('# An example of this file can be found in the TestCases directory.\n')
+    file.write('#\n')
+    file.write('# Author: Ettore Saetta, Lorenzo Russo, Renato Tognaccini.\n')
+    file.write('# Theoretical and Applied Aerodynamic Research Group (TAARG),\n')
+    file.write('# University of Naples Federico II\n')
+    file.write('# -------------------------------------------------------------------------------------\n')
+    file.write('#\n')
+    file.write('MARKER_ACTDISK= \n')
+    file.write('CENTER= \n')
+    file.write('AXIS= \n')
+    file.write('RADIUS= '+str(R)+'\n')
+    file.write('ADV_RATIO= '+str(J)+'\n')
+    file.write('NROW= '+str(Stations)+'\n')
+    file.write('# rs=r/R        dCT/drs       dCP/drs       dCR/drs\n')
+
+    for i in range(0, Stations):
+        file.write(f'  {r[i]:.7f}     {CTrs[i]:.7f}     {CPrs[i]:.7f}     0.0\n')
+
+    file.close()
+
+##########################
+###        Main        ###
+##########################
+# Screen output
+print('------------------ Optimal Propeller vsn 1.0.0 ------------------')
+print('| Computation of the optimal dCT/dr and dCP/dr distributions.   |')
+print('| Based on the inviscid theory of the optimal propeller.        |')
+print('|                                                               |')
+print('| This code is used to generate the actuator disk input data    |')
+print('| file needed for the VARIABLE_LOAD actuator disk type          |')
+print('| implemented in SU2 7.0.7.                                     |')
```

Comment:
I think the version of any component of SU2 should simply be the version of SU2.

Reply:
Version changed. I put 7.0.7 because it should be in the next release. Is it good or should I write 7.0.6?

Reply:
I don't know if the next release will be 7.0.7 or 7.1.0.
Make it 7.0.6 please, there is a script that will update all 7.0.6's to the correct version before the release.

Reply:
ok, I didn't know about it. Version changed to 7.0.6.